cranserver/lib/registry.py

```diff
@@ -10,7 +10,7 @@ class DuplicatePkgException(Exception): pass
 class Registry(dict):
 
     def __init__(self, storage=None):
-        if storage:
+        if storage is not None:
```

Comment:
I hat to change this because TIL, when using `if object`, Python will actually check the `__len__` of the object as a factor. If the length of the object is 0, then it evaluates to `False`. Since the storage object has no packages on init it would always evaluate to `False`. Hence the change here.